resource/history/magento/product-community-edition/2.4.7-p3.json

```
@@ -0,0 +1,8 @@
+{
+    "require": {
+      "magento/inventory-metapackage": "1.2.7-p3",
+      "magento/page-builder": "1.7.4-p3",
+      "magento/security-package": "1.1.6-p3",
+      "adobe-commerce/os-extensions-metapackage": "~1.0"
```

```suggestion
      "adobe-commerce/os-extensions-metapackage": "~1.0",
      "magento/inventory-metapackage": "1.2.7-p3",
      "magento/page-builder": "1.7.4-p3",
      "magento/security-package": "1.1.6-p3"
```

same thing about ordering